ui/src/common/layouts/SearchLayout/SearchLayout.jsx

```diff
@@ -32,12 +32,12 @@ class SearchLayout extends Component {
     } = this.props;
     return (
       <Row className="__SearchLayout__" gutter={32} type="flex" justify="start">
-        <Col xs={0} lg={8} xl={6} xxl={5}>
+        <Col xs={0} lg={8} xl={7} xxl={5}>
           {!withoutAggregations && (
             <ResponsiveView min="lg" render={() => this.renderAggregations()} />
           )}
         </Col>
-        <Col xs={24} lg={16} xl={15} xxl={14}>
+        <Col xs={24} lg={16} xl={16} xxl={14}>
```

Comment:
How about embedded search ? It may need to be changed too.

Reply:
it looks ok in the embed search